src/status_im/ui/screens/wallet/main/styles.cljs

```diff
@@ -3,19 +3,28 @@
   (:require [status-im.components.styles :as styles]
             [status-im.utils.platform :as platform]))
 
-(def error-container
+;;;;;;;;;;;;;;;;;;
```

Comment:
a lot of noise no?

Reply:
there was already one title like this for the wallet part I just added those to clarify the other parts. I'm neutral on that one I like that it makes a visible and readable separation but yeah it is noisy